React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm

```diff
@@ -401,6 +407,8 @@ - (void)prepareForRecycle
   _shouldUpdateContentInsetAdjustmentBehavior = YES;
   _state.reset();
   _isUserTriggeredScrolling = NO;
+  _oldFrame = self.frame;
+  [self setFrame:CGRectZero];
```

Comment:
Could you try to replace this part with this code:

```objc
  CGRect oldFrame = self.frame;
  self.frame = CGRectZero;
  self.frame = oldFrame;
```

and remove all the other changes. It should work and it save us from keeping a state variable which can get out of sync easily.

Reply:
Brilliant. I should've thought of that 👍